src/components/ProblemsTable.tsx

```diff
@@ -92,7 +91,37 @@ export const ProblemsTable = ({
         return context?.focusedIssues.some((issue) => bannerLevel === issue);
       });
 
-    return reports;
+    // **FIX referenced in own type annotation
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const expandableReports: any = {};
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
can't we move this new way to return a report to a separate function or even file if needed?